app/src/main/java/com/endava/myendava/fragments/ProfileFragment.java

```diff
@@ -95,7 +87,6 @@
     private Unbinder mUnbinder;
 
     public ProfileFragment() {
-        // required empty public constructor
```

Comment:
there is a reason for removing the // required empty constructor? 🤔

Reply:
No